include/crypto/asn1.h

```diff
@@ -70,7 +70,8 @@ struct evp_pkey_asn1_method_st {
      */
     /* Exports to providers */
     size_t (*dirty_cnt) (const EVP_PKEY *pk);
-    void *(*export_to) (const EVP_PKEY *pk, EVP_KEYMGMT *keymgmt);
+    void *(*export_to) (const EVP_PKEY *pk, EVP_KEYMGMT *keymgmt,
```

Comment:
@levitte sorry to come late with this comment, but would it be possible to document the expected behaviour of an `export_to` function?

I know it's an internal function, but being an internal "interface" it warrants a bit of care even with things still changing (or especially because of this).

I encountered this while working on #10631 and noticed only now that both `dirty_cnt` and `export_to` are underdocumented and the only option right now is to `git blame` to see in which PR they were added, read through the discussions, and reverse engineer what the `dh` and `dsa` implementations currently do.

Reply:
It's there.  `perldoc -o man doc/man3/EVP_PKEY_ASN1_METHOD.pod`, then scroll down a bit.